src/examples/tools/materialphysicalproperties/slags.ipynb

```diff
@@ -55,11 +55,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
It looks like you saved the Jupyter notebook with output. You can clear the output and then save it.